File: scripts/py/3.py
import time
import requests
import math

reqUrl="https://unpkg.com/@surely-vue/table@4.3.13/dist/index.umd.js"

res = requests.get(reqUrl).text

length = len(res)

count = math.floor(length/15000)

my_list=[]

for i in range(count):
    my_list.append(res[i*15000:(i+1)*15000])
    time.sleep(1)
    print(my_list[i])

File: scripts/js/3.js
const url = "https://unpkg.com/@surely-vue/table@4.3.13/dist/index.umd.js";

fetch(url)
  .then((res) => res.text())
  .then((res) => {
    globalThis.arr = [];
    let count = Math.floor(res.length / 15000),
      i = 0;
    while (i < count) {
      arr.push(res.slice(i * 15000, 15000 * (i + 1)));
      i++;
    }
  })
  .then(() => {
    console.log(arr.length);
  });
